partner_priority/models/partner_priority.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Patrick Wilson <patrickraymondwilson@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
+
+
+class PartnerPriority(models.Model):
+    _name = 'partner.priority'
+    _order = 'sequence'
+    _description = 'Partner Priority'

```

Comment:
Could you improve the description, please?

Reply:
I'm not sure what to change this to, do you have a suggestion you would like to see instead of what's there?

Reply:
Maybe the model reason

Reply:
I'm sorry, I just don't follow what you are looking for. I feel this is a fairly basic module and the description is stating exactly what it is, priority to the partners, but maybe i'm mis-understanding this too.

Reply:
👍